# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../unionfs/no-depmod.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We must not touch the hosts /lib/modules/...

  - Rene Rebe <rene@exactcode.de>

--- unionfs-1.1.2/Makefile.orig	2006-01-25 22:58:11.000000000 +0100
+++ unionfs-1.1.2/Makefile	2006-01-29 14:23:10.000000000 +0100
@@ -101,7 +101,6 @@
 install-mod: unionfs.ko
 	mkdir -p ${MODPREFIX}/${MODDIR}/kernel/fs/unionfs
 	cp unionfs.ko ${MODPREFIX}/${MODDIR}/kernel/fs/unionfs
-	-/sbin/depmod -a
 
 install: install-utils install-mod
 
